numpy/testing/utils.py

```diff
@@ -255,6 +255,7 @@ def assert_equal(actual,desired,err_msg='',verbose=True):
      DESIRED: 6
 
     """
+    __tracebackhide__ = True # Hide traceback for py.test
```

Comment:
PEP8 [says](https://www.python.org/dev/peps/pep-0008/#inline-comments) (at least) 2 spaces between end of line and opening `#` for inline comments. Same applies to all other modified lines.